src/example_scripts/example_submission.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3
+"""
+Overview
+========
+This script leverages the "robot submissions" feature which allows someone to make
+unlimited submissions to a competition. An example use case is the ChaGrade teaching
+website: student submissions are made and scores returned via robot submissions.
+
+Usage
+=====
+
+1. Create a competition that allows robots, and create a user marked as a robot
+   user. Use that username and password below.
+
+2. Get into a python3 environment with requests installed
+
+3. Review this script and edit the applicable variables, like...
+
+    CODALAB_URL
+    USERNAME
+    PASSWORD
+    ...
+
+4. Then execute the contents of this script:
+
+    ./example_submission.py
+"""
+# ----------------------------------------------------------------------------
+# Configure these
+# ----------------------------------------------------------------------------
+CODALAB_URL = 'http://localhost/'
+USERNAME = 'admin'
+PASSWORD = 'admin'
+PHASE_ID = None
+SUBMISSION_ZIP_PATH = '../tests/functional/test_files/submission.zip'
+
+
+# ----------------------------------------------------------------------------
+# Script start..
+# ----------------------------------------------------------------------------
+from urllib.parse import urljoin  # noqa: E402
+import requests                   # noqa: E402,E261  # Ignore E261 to line up these noqa
+
+
+# Login
+login_url = urljoin(CODALAB_URL, '/api/api-token-auth/')
+resp = requests.post(login_url, {"username": USERNAME, "password": PASSWORD})
+if resp.status_code != 200:
+    print(f"Failed to login: {resp.content}")
+    exit(-1)
+
+# Setup auth headers for the rest of communication
+token = resp.json()["token"]
+headers = {
+    "Authorization": f"Token {token}"
+}
+
+# Check if we can make a submission
+can_make_sub_url = urljoin(CODALAB_URL, f'/api/can_make_submission/{PHASE_ID}/')
+resp = requests.get(can_make_sub_url, headers=headers)
+
+if not resp.json()['can']:
+    print(f"Failed to create submission: {resp.json()['reason']}")
```

Comment:
```suggestion
    print(f"You are not allowed to make a submission: {resp.json()['reason']}")
```

Reply:
I figured it made more sense as "Failed to create submission" so the resp.json()['reason'] isn't so redundant.

For example:
Failed to create submission: User not approved to participate in this competition
VS
You are not allowed to make a submission: User not approved to participate in this competition

Reply:
great point, sounds fine then!